src/components/Threads.vue

```diff
@@ -43,6 +48,10 @@ export default {
     displayThread (thread) {
       window.scrollTo(0, 0)
       this.displayedThread = thread
+    },
+    onLoad (index, done) {
+      if (this.displayedThread) return
```

Comment:
I don't know what `displayedThread` is... I feel like it's doing two things and once and there should be two different routes here.
The "smell" I'm seeing is that infinite loading is not at all relevant to the case where you are 'displayThread'?

Are they two different components / pages?
- ThreadsView
- ThreadView